cmd/registry/cmd/upload/artifact.go

```diff
@@ -124,7 +124,7 @@ func buildDisplaySettingsArtifact(ctx context.Context, jsonBytes []byte) (*rpc.A
 	}
 	return &rpc.Artifact{
 		Contents: artifactBytes,
-		MimeType: patch.DisplaySettingsMimeType,
+		MimeType: patch.MimeTypeForKind("DisplaySettings"),
```

Comment:
This is admittedly a bit uglier, but is because further down, in the patch package, all the information about supported artifact types is consolidated into one place. I didn't (in this PR) try to simplify this file further, but I expect that it can be refactored similarly to patch/artifact.go to remove these type-specific build functions.